README.md

```diff
@@ -584,7 +585,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Honey Patel](https://github.com/honeykpatel)
   - [Hrithik Chauhan](https://github.com/Hrithik5)
   - [Hulya Karakaya](https://github.com/hulyak)
-
```

Comment:
Please undo the line deleted here.

Reply:
First I added my name and github link here, @kaiwalyakoparkar  asked me to enter it alphabetically. It was on other branch. I made other branch in this branch i removed the line which i entered last and paste it on top second number and make a new pull request.

Reply:
Tell me what is causing the issue so i can fix it.

Reply:
Your profile addition looks good. Just add the line you removed. You can go to the current branch, add that line again `587` and commit it. PR will auto fetch those changes.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
done